package.json

```diff
@@ -226,22 +226,22 @@
         },
         {
           "command": "java.project.addLibraries",
-          "when": "view == javaDependencyExplorer && viewItem =~ /java:container\/referenced-libraries$/",
+          "when": "view == javaDependencyExplorer && viewItem =~ /java:container/referenced-libraries$/",
```

Comment:
looks like this is a regular pattern. Is there any reason to remove the escape char `\/` ?

Reply:
My bad, `npm install` automatically removes them